include/frozen/bits/basic_types.h

```diff
@@ -40,7 +41,7 @@ struct ignored_arg {};
 template <class T, std::size_t N>
 class cvector {
   T data_[N] = {}; // zero-initialization for scalar type T, default-initialized otherwise
-  std::size_t dsize_ = 0;
+  FROZEN_SIZE_T dsize_ = 0;
 
   template <class Iter>
   constexpr cvector(Iter iter, size_t size)
```

Comment:
Do we need to add a static assertion in the constructor to make sure that `sizeof(FROZEN_SIZE_T) > 4 || N <= (1U << 31)`?